$NetBSD: patch-configure.ac,v 1.1 2012/03/21 10:07:36 wiz Exp $

Use pkg-config file for lrdf, fixes its detection for >= 0.5.0.

--- configure.ac.orig	2007-11-26 22:23:42.000000000 +0000
+++ configure.ac
@@ -193,22 +193,21 @@ AC_ARG_ENABLE(lrdf,
   esac
 ])
 
-if test x"$jr_enable_lrdf" = x"yes"; then
-  AC_CHECK_HEADER(lrdf.h,
-    AC_CHECK_LIB(lrdf, lrdf_init, LRDF_FOUND="yes", LRDF_FOUND="no"),
-    LRDF_FOUND="no")
+if test "$jr_enable_lrdf" = "yes"; then
+  PKG_CHECK_MODULES(LRDF, lrdf, LRDF_FOUND="yes", LRDF_FOUND="no")
+
+  if test "$XML_FOUND" = "yes"; then
+    AC_DEFINE(HAVE_LRDF, 1, "whether or not lrdf support is enabled")
+    AC_SUBST(LRDF_CFLAGS)
+    AC_SUBST(LRDF_LIBS)
+  else
+    AC_MSG_WARN([lrdf support is disabled])
+  fi
 else
+  XML_FOUND="no"
   AC_MSG_WARN([lrdf support is disabled])
-  LRDF_FOUND="no"
 fi
 
-if test x$LRDF_FOUND = xyes; then
-  AC_DEFINE(HAVE_LRDF, 1, "whether or not lrdf support is enabled")
-  LRDF_CFLAGS=""
-  LRDF_LIBS="-llrdf"
-  AC_SUBST(LRDF_CFLAGS)
-  AC_SUBST(LRDF_LIBS)
-fi
 AM_CONDITIONAL(HAVE_LRDF, test "$LRDF_FOUND" = "yes")
 
 
